openqa-trigger-bisect-jobs

```diff
@@ -47,34 +50,49 @@ def call(cmds, dry_run=False):
 def openqa_clone(cmds, dry_run, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
     return call(['openqa-clone-job'] + default_opts + cmds + default_cmds, dry_run)
 
+def fetch_url(url, request_type='text'):
+    try:
+        content = requests.get(url.geturl())
+        content.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        log.error("Error while fetching "+url.geturl()+": "+str(e))
```

Comment:
```suggestion
        log.error("Error while fetching " + url.geturl() + ": " + str(e))
```
It would be nice to use spaces in other occurrences as well.

Reply:
done (I'm using `%` now)